bus-mapping/src/state_db.rs

```diff
@@ -59,12 +59,22 @@ impl Account {
             code_hash: *CODE_HASH_ZERO,
         }
     }
+
+    /// Return if account is empty or not.
+    pub fn is_empty(&self) -> bool {
```

Comment:
Ah, this is used [here](https://github.com/han0110/zkevm-circuits/blob/feature%2Fcall/bus-mapping/src/evm/opcodes/call.rs#L126) in #278 and I forgot to remove it. Should I remove it?

Reply:
Thanks! No need to remove it, I just wanted to understand where it was needed because I was thinking of a possible edge case but it may not apply here.  The edge case I was thinking about is that an Account has a Storage which we model as a HashMap.  But keys can be deleted in EVM by setting the value to 0, but our HashMap will not be empty in that case.  Maybe this edge case never occurs, because to write to the Storage you would need a code hash != empty.

Nevertheless let me confirm this: `is_empty` should return true when either the account never existed, or it has been destroyed, and no other case right?